packages/server/graphql/public/types/User.ts

```diff
@@ -315,8 +315,8 @@ const User: ReqResolvers<'User'> = {
 
   isConnected: async ({id: userId}) => {
     const redis = getRedis()
-    const connectedSocketsCount = await redis.llen(`presence:${userId}`)
-    return connectedSocketsCount > 0
+    const socketCount = await redis.get(`awareness:${userId}`)
+    return Number(socketCount) > 0
```

Comment:
+1 instead of separate bookkeeping we could just count if they're subscribed to their notification channel
```
    const redis = getRedis()
    const channelName = `${SubscriptionChannel.NOTIFICATION}.${userId}`
    const [_, subscribers] = await redis.pubsub('NUMSUB', channelName)
    return subscribers > 0
```